### ISSUE TITLE
CLEAN software not talking to database?  

### ISSUE THREAD

**Issue by JulieJager** [syncids/syncids-ui#619]
@ntgn81 I have a key piece of migration software that is not working.  It has not been working properly for quite some time.  We've had a time-consuming work-around but now we have a huge migration where the work-around may cause problems/mistakes.

PROCEDURE:
Log into Home 2 via user@syncids.com
Create client: ITL
Create matter: Upload
Create foreign reference: 1234
Upload "OpenMatters....txt" to open several matters at once
Upload "SyncIDSMigration...Bronze.txt" to clean India's data

WHAT HAPPENS:
user@syncids.com receives an email with a cleaned up version of India's text file.  CLEAN auto-formats India's data and puts in into a computer readable format, finds errors, etc.  It also creates a Matter Line associated with the data (Client;Matter;Application#;ArtUnit;FilingDate;Open/Closed.

PROBLEM:  
The latter stage is not happening.  I cannot get the cleaned-up version when uploading India's data.  India's data is different from our other uploads in that it requires a communication with the matterdb in order to create Matter Lines.  I can confirm the Matters are created in db but India's data just isn't grabbing the data.     

 POSSIBILITIES:  Jim suspects either Clean is not looking into the right database for the Matter information, or CLEAN is pointing to the wrong DNS, or the spacing is off in India's text file.  The ApplicationNumber is the connector from India's text file to the database.  I confirmed the spacing is no different than it used to be (application number starts at position 53).

BOTTOM LINE:  I cannot review the code associated with CLEAN to confirm/deny any of the above.  

Can you?  Gosh, I hope this makes sense.  

Below is a test case.  FirmName = 'syncidscom' and ClientNumber = 'ITL' and MatterNumber = '3022US'

[SyncIDSMigration20160512 (bronzetest).txt](https://github.com/syncids/syncids-ui/files/540522/SyncIDSMigration20160512.bronzetest.txt)

**Comment by ntgn81** [comment]
@JulieJager , can you give it another try, please? There was a typo on upload script, I'm not sure how new it is.

**Comment by JulieJager** [comment]
OMG.  It works!  You have no idea how happy I am (well, how happy Deb will be).  Thank you!

**Comment by ntgn81** [comment]
Yay
